xandikos/access.py

```diff
@@ -22,7 +22,6 @@
 See http://www.webdav.org/specs/rfc3744.html
 """
 
-from defusedxml.ElementTree import fromstring as xmlparse
```

Comment:
If this isn't done before importing xml.etree then we can end up with multiple different ElementTree implementations, which leads to TypeErrors. 

(we should probably have a central ElementTree import somewhere that imports from defusedxml first and is used by all other parts of the code, so these two lines aren't repeated all over the place)